Updates week 1

mcvholloway opened the pull request:
This PR includes the updates to the README. The last question for the presentation is still the rough draft version, but I think we were both happy with the first two questions.
Updates for week 1:
Shortened the first public art notebook; we had run just a little short on time with metro and it felt like we were always slightly running behind last year, so shifting the end section to week 2 seems like a good choice. 
Subbed in hospitals data for week 1 coding tasks.

mcvholloway committed:
updated README

mcvholloway committed:
updated week 1 slides

mcvholloway committed:
Shorten public art notebook

The reason for this change is that the previous version was slightly
too long to allow time to finish and let student start working on the
associated practice notebook. The more complex filtering will be moved
to the week 2 materials.

mcvholloway committed:
Replace old week one coding tasks

This commit replaces last year's week one coding task with one where
the students will explore a dataset of US hospitals. When working on
the overall project, they can incorporate some of their findings from
this exercise or expand on it

raom1 reviewed (approved):
Looks good. I like shortening the public art notebook

mcvholloway committed:
Add upstream to week 1 materials

This will allow students to pull in the new week's materials from the original repository at the beginning of each class.

mcvholloway committed:
Rename public art notebook

mcvholloway committed:
Rename public art notebook

mcvholloway committed:
Remove week 1 pdf

mcvholloway committed:
Update week 1 slides

mcvholloway committed:
Update git/GitHub explanation
